pkgs/applications/science/misc/cytoscape/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchurl, jre, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "cytoscape-${version}";
+  version = "3.4.0";
+
+  src = fetchurl {
+    url = "http://chianti.ucsd.edu/${name}/${name}.tar.gz";
+    sha256 = "065fsqa01w7j85nljwwc0677lfw112xphnyn1c4hb04166q082p2";
+  };
+
+  buildInputs = [jre makeWrapper];
+
+  installPhase = ''
+    mkdir -pv $out/{share,bin}
+    cp -Rv * $out/share/
+
+    cat > $out/bin/cytoscape << EOF
```

Comment:
Shouldn't a symlink be enough? The hashbang will get replaced in final derivation.